### DISCUSSION TITLE
eBus to USB Adapter for 1TE DIN Rail Module

### DISCUSSION THREAD

**Discussion by jkyprian** [john30/ebusd#753] [General]
Anyone here interested in an eBus adapter for a 1TE DIN rail module?

I created it before the COVID breakout and hadn't time, until recently, to solder and test it. It uses a series transistor regulator circuit with a ZRC400 to extract the operating voltage from the eBus, an ADuM1201 for the galvanic isolation, and a CH330 for the uart to USB conversion.

![ebus2usb](https://user-images.githubusercontent.com/1710378/206872461-53b793ed-a38b-4d04-8466-6635eeeee8ef.jpg)

**Comment by stevenbrookes** [comment]
Yes this looks very interesting to me as it would lead to a neat install with my solar/heatpump installation.  Are you looking to sell these or want a tester ;)

**Reply by jkyprian** [reply]
I'm considering publishing the design and was wondering how much interest there is.

**Comment by jkyprian** [comment]
KiCad schematic, pcb layout, and production files for JLCPCB are now available [here](https://github.com/jkyprian/ebus2usb).

![ebus2usb-img-v0 8](https://user-images.githubusercontent.com/1710378/236695912-eef0f115-281a-4003-a591-33a5c4510160.jpg)
